test/verify/check-pages

```diff
@@ -1009,6 +1009,30 @@ OnCalendar=daily
         self.assertEqual(str(user_info["id"]), m.execute("id -u admin").strip())
         self.assertEqual(str(user_info["gid"]), m.execute("id -g admin").strip())
 
+    @testlib.skipImage("No SELinux", "debian-*", "ubuntu-*", "arch")
+    def testFSReplaceSELinuxContext(self) -> None:
+        b = self.browser
+        m = self.machine
+
+        path = "/tmp/custom-selinux-context"
```

Comment:
Please either clean up the file, or put it into `self.vm_tmpdir` (probably better)